public/app/core/components/AppChrome/Organization/OrganizationSwitcher.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { OrgRole } from '@grafana/data';
+import { configureStore } from 'app/store/configureStore';
+import * as appTypes from 'app/types';
+
+import { OrganizationSwitcher } from './OrganizationSwitcher';
+
+jest.mock('app/features/org/state/actions', () => ({
+  ...jest.requireActual('app/features/org/state/actions'),
+  getUserOrganizations: jest.fn(),
+  setUserOrganization: jest.fn(),
+}));
+
+jest.mock('app/types', () => ({
+  ...jest.requireActual('app/types'),
+  useDispatch: () => jest.fn(),
+}));
+
+const renderWithProvider = ({ initialState }: { initialState?: Partial<appTypes.StoreState> }) => {
+  const store = configureStore(initialState);
+
+  render(
+    <Provider store={store}>
+      <OrganizationSwitcher />
+    </Provider>
+  );
+};
+
+describe('OrganisationSwitcher', () => {
+  it('should only render if more than one organisations', () => {
+    renderWithProvider({
+      initialState: {
+        organization: {
+          organization: { name: 'test', id: 1 },
+          userOrgs: [
+            { orgId: 1, name: 'test', role: OrgRole.Admin },
+            { orgId: 2, name: 'test2', role: OrgRole.Admin },
+          ],
+        },
+      },
+    });
+
+    expect(screen.getByRole('combobox')).toBeInTheDocument();
```

Comment:
we should use our `aria-label` to select the `combobox` here and in the rest of the tests:
`expect(screen.getByRole('combobox', { name: 'Select organization' }).toBeInTheDocument();`